package.json

```diff
@@ -31,7 +31,9 @@
     "strip-ansi-cli": "^2.0.0"
   },
   "dependencies": {
+    "ascii-table": "0.0.9",
     "bn": "^1.0.5",
+    "bn.js": "^5.1.1",
```

Comment:
I'm actually not sure what bn above is for?... @janedegtiareva you added it here https://github.com/nearprotocol/near-shell/commit/5ad3a891d5d1b6240df78dc87d285f5c6a290c68 but this is not `bn.js`.

Reply:
This was most likely refactored out at some point. I don't see any usages, so we can remove the other one. I'll put together a cleanup PR